josharagon/Refactor-Tractor
# Fixes/resolve merge.conflicts

## Pull Request

<br>

### Description

* Is this a feature or a fix?

- Updates the UI with the same features provided by branch feature/update-ui
- Resolves merge conflicts in scripts.js, deletes unused code and tidies up codebase
- Adds additional styling and accessibility to charts.
- Standardizes week retrieval method across Classes, adjusts unit tests accordingly.

* Where should the reviewer start?

- Scripts.js was completely reorganized, but none of the core functionality should be altered.
- Some changes made to Activity-test, sleep-test and hydration-test

* What functionalities do these changes effect? Why were these changes made? What was the motivation behind these changes?

- All data on the UI should render correctly including charts and other displays. Functionality should be unchanged, code base was just tidied up and merge conflicts resolved.

* What outside features or research did you use to implement this fix?

None.

* How should this be tested?

Refresh the page multiple times, cross-check displayed data with dev tools. Press any key to run a POST request, verify that data is posted correctly.

<br>

***

#### Please review considerations below:
